api/v1alpha1/azurevalidator_types.go

```diff
@@ -27,11 +27,14 @@ type AzureValidatorSpec struct {
 	// +kubebuilder:validation:MaxItems=5
 	// +kubebuilder:validation:XValidation:message="RBACRules must have unique names",rule="self.all(e, size(self.filter(x, x.name == e.name)) == 1)"
 	RBACRules []RBACRule `json:"rbacRules" yaml:"rbacRules"`
-	Auth      AzureAuth  `json:"auth" yaml:"auth"`
+	// Rules for validating that images exist in an Azure Compute Gallery published as a community
+	// gallery.
+	CommunityGalleryImageRules []CommunityGalleryImageRule `json:"communityGalleryImageRules" yaml:"communityGalleryImageRules"`
```

Comment:
Now that the Azure plugin supports more than one rule type, please add `omitempty` to both rule types.